streamwebs_frontend/streamwebs/models.py

```diff
@@ -1,5 +1,42 @@
 from __future__ import unicode_literals
 
+import datetime
+
 from django.db import models
+from django.contrib.auth.models import User
+from django.core.exceptions import ValidationError
+from django.utils.translation import ugettext_lazy as _
 
 # Create your models here.
+
+def validate_UserProfile_school(profile):
+    if not profile.school in dict(profile.SCHOOL_CHOICES):
+        raise ValidationError('That school is not in the list.')
+
+def validate_UserProfile_birthdate(profile):
+    today = datetime.datetime.now() 
+    if profile.birthdate.year > today.year - 13:
+        raise ValidationError('You must have been born before %s' % (today.year-13))
+    elif profile.birthdate.year == today.year - 13:
+        if profile.birthdate.month > today.month: 
+            raise ValidationError('You are not yet 13 (month)')
+        elif profile.birthdate.month == today.month:
+            if profile.birthdate.day > today.day:
+                raise ValidationError('You are not yet 13 (days)')
+            
+class UserProfile(models.Model):
+    SCHOOL_A = 'a'
+    SCHOOL_B = 'b'
+    SCHOOL_C = 'c'
+    SCHOOL_CHOICES = (
+      (SCHOOL_A, 'School A'),
+      (SCHOOL_B, 'School B'),
+      (SCHOOL_C, 'School C'),
+    )
```

Comment:
I think at some point before this gets merged, you need to abstract the schools out of the tests - maybe have the model get the list of schools from settings or from a schools table, then have your test explicitly over-ride that setting or add a school to the table for the test